test/elf/section-order.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 . $(dirname $0)/common.inc
 
-# qemu crashes if the ELF header is not mapped to memory
-[ -z "$QEMU" ] || skip
```

Comment:
I'm confused with this change. This line prevents the rest of the script to be executed if $QEMU is non-empty. In other words, this test will be skipped if it is running with QEMU. Isn't it the opposite of what you are trying to do?